corehq/ex-submodules/casexml/apps/case/models.py

```diff
@@ -356,6 +352,12 @@ def get_case_property(self, property):
         except Exception:
             return None
 
+    def get_closing_transactions(self):
+        return filter(lambda action: action.action_type == const.CASE_ACTION_CLOSE, reversed(self.actions))
```

Comment:
modelled after this: https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/models.py#L359

presumably because close actions are more likely to be at the end of the list